annotations/src/main/java/com/statful/client/annotations/Timer.java

```diff
@@ -38,4 +38,9 @@
      * @return An array of aggregations to send to Statful
      */
     Aggregation[] aggregations() default {};
+
+    /**
+     * @return An int of the sampleRate to send to Statful
+     */
+    int sampleRate() default 0;
```

Comment:
What if I want to disable actually sending that specific timer metric for some reason? What if removing the annotation for that purpose is not an option because I have a custom aspect around @Timer that I still want to execute? If you give me an option to set a specific sample rate, I will want to use it and expect that it would work the way I want it to...